### ISSUE TITLE
Sort users by name, email, last login, etc. - also view all users on one page instead of multiple pages

### ISSUE THREAD

**Issue by EricSchroeder1** [metabase/metabase#19738]
**Is your feature request related to a problem? Please describe.**
For productive work, it would be great if we could sort the list of users in the `/admin/people` section. For example, alphabetically by name, email, groups last login, ... (ascending and descending).

**Describe the solution you'd like**
Click on e.g. the section `Name` and automatically sort the contents of the column. 

**Describe alternatives you've considered**
A drop down field would also be conceivable. Displaying all `Metabase` users on one page would also facilitate many processes - currently only 25 users are displayed per page - maybe just set a checkbox?

**How important is this feature to you?**
This feature would currently decrease manual workload.

**Comment by flamber** [comment]
Very related to #14789 and #8126

**Comment by EricSchroeder1** [comment]
@flamber yeah, but https://github.com/metabase/metabase/issues/8126 was created in 2018. Also it was added to the [0.40 milestone](https://github.com/metabase/metabase/milestone/131), but metabase does not yet have this feature, see [04-managing-users](https://github.com/metabase/metabase/blob/master/docs/administration-guide/04-managing-users.md).

**Comment by flamber** [comment]
@EricSchroeder1 Creating multiple issue will not mean anything gets fixed or implemented quicker - just the opposite.
It was never on the 0.40 milestone. Such information will be in the history of the issue.
And the issue is still open, so it has not been implemented/fixed. Once an issue is closed, then it will get a milestone attached.
